media/js/mozorg/home/takeover-2016.js

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
+    'use strict';
+
+    // Array.prototype.indexOf is undefined in older versions of browsers
+    var arrayIndexOfSupported = typeof Array.prototype.indexOf === 'function';
+
+    var $body = $('body');
+    var $takeoverModal = $('#fundraising_takeover');
+    var $overlay = $('.page-overlay');
+    var $closeModal = $('#close_takeover');
+    var $form = $('#fundraising-form');
+    var $donateOptionsSection = $('.donate-options');
+    var $donateCustom = $('#donate-custom');
+    var $optionLabels = $('label[class!="own-amount"]', $donateOptionsSection);
+    var $options = $('input[type="radio"]', $donateOptionsSection);
+
+    if ('Intl' in window && 'NumberFormat' in Intl && $takeoverModal.length > 0) {
```

Comment:
Instead of putting this inline at the top of the file, suggest making it a function which is then called from inside `initTakeOver()`?